.github/workflows/testBuild.yml

```
@@ -0,0 +1,182 @@
+        env:
+          S3_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          S3_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          S3_BUCKET: ad-hoc-expensify-cash
+          S3_REGION: us-east-1
+          PULL_REQUEST_NUMBER: ${{ github.event.number }}
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: ios
+          path: ./ios_paths.json
+
+  # web:
+  #   name: Build and deploy Web
+  #   needs: validateActor
+  #   if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+  #       with:
+  #         fetch-depth: 0
+  #         ref: ${{ github.event.pull_request.head.sha }}
+
+  #     - uses: Expensify/App/.github/actions/composite/setupNode@main
+
+  #     - name: Configure AWS Credentials
+  #       # Version: 1.5.5
+  #       uses: aws-actions/configure-aws-credentials@e97d7fbc8e0e5af69631c13daa0f4b5a8d88165b
+  #       with:
+  #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  #         aws-region: us-east-1
+
+  #     - name: Build web for staging
+  #       run: npm run build-staging
+
+  #     - name: Build docs
+  #       run: npm run storybook-build
+  #       continue-on-error: true
+
+  #     - name: Deploy to S3 for internal testing
+  #       run: aws s3 cp --recursive --acl public-read "$GITHUB_WORKSPACE"/dist s3://ad-hoc-expensify-cash/web/"$PULL_REQUEST_NUMBER"
+  #       env:
+  #         PULL_REQUEST_NUMBER: ${{ github.event.number }}
+
+  postGithubComment:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
+    needs: [android, ios]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - uses: actions/download-artifact@v3
+
+      - name: Read JSONs with paths
+        id: set_var
+        run: |
+          content_android="$(cat ./android/android_paths.json)"
+          content_android="${content_android//'%'/'%25'}"
+          content_android="${content_android//$'\n'/'%0A'}"
+          content_android="${content_android//$'\r'/'%0D'}"
+          echo "android_paths=$content_android" >> "$GITHUB_OUTPUT"
+          content_ios="$(cat ./ios/ios_paths.json)"
+          content_ios="${content_ios//'%'/'%25'}"
+          content_ios="${content_ios//$'\n'/'%0A'}"
+          content_ios="${content_ios//$'\r'/'%0D'}"
+          echo "ios_paths=$content_ios" >> "$GITHUB_OUTPUT"
+
+      - name: Publish links to apps for download
+        run: |
+          gh pr comment --body \
+          "| android :robot:  | iOS :apple: | 
+          | ------------- | ------------- |
+          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  |"
```

I think that we should add some comment explaining what these links are.
```suggestion
          "Use the links below to test this build in android and iOS. Happy testing!
          | android :robot:  | iOS :apple: | 
          | ------------- | ------------- |
          | ${{fromJson(steps.set_var.outputs.android_paths).html_path}}  | ${{fromJson(steps.set_var.outputs.ios_paths).html_path}}  |"
```